lib/install/config/assets.js

```diff
@@ -0,0 +1,40 @@
+// Handles static asset integration - like sass and images
+// Note: You must restart bin/webpack-watcher for changes to take effect
+
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const process = require('process')
+const sharedConfig = require('./shared.js')
+const { devServer } = require('../../package.json')
+
+const production = process.env.NODE_ENV === 'production'
+const hotServerAddr = `http://${devServer.host}:${devServer.port}/`
```

Comment:
This is already available in `config.x.webpacker[:dev_server_host]` and we already have the pattern of using `process.env` to read those configs with `WEBPACK_DIST_DIR` when running `webpacker:compile`. Why not just follow that approach here for the dev server address?

Also, down below on `publicPath`, `WEBPACK_DIST_DIR` can be used as well. 

This will mean that the binstub generated needs to insert those environment variables but that should be a one-time thing that is never updated once established.